components/threading/channel_threads/thread_footer/thread_footer.tsx

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/* eslint-disable react/prop-types */
+
+import React, {memo, ComponentProps} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import './thread_footer.scss';
+
+import Avatars from 'components/widgets/users/avatars';
+
+import Button from 'components/threading/common/button';
+import FollowButton from 'components/threading/common/follow_button';
+
+import Timestamp from 'components/timestamp';
+import SimpleTooltip from 'components/simple_tooltip';
+import ReplyIcon from 'components/widgets/icons/reply_icon';
+
+import {THREADING_TIME} from '../../common/options';
+
+type Props = {
+    participants: ComponentProps<typeof Avatars>['users'];
+    totalReplies: number;
+    newReplies: number;
+    lastReplyAt: ComponentProps<typeof Timestamp>['value'];
+    isFollowing: boolean;
+    actions: {
+        follow: () => void,
+        unfollow: () => void,
+        requestOpenThread: () => void;
+    }
+};
+
+function ThreadFooter({
+    participants,
+    totalReplies = 0,
+    newReplies = 0,
+    lastReplyAt,
+    isFollowing,
+    actions: {
+        follow,
+        unfollow,
+        requestOpenThread: open,
+    },
+}: Props) {
+    return (
+        <div className='ThreadFooter'>
+            {Boolean(newReplies) && <>
```

Comment:
Why the Fragment (`<>`) for a single component?

Reply:
I think a remnant of an earlier version, will change 👍